deployment/src/strongmind_deployment/autotag.py

```diff
@@ -67,22 +115,10 @@ def auto_tag(args, auto_tags):
             args.props["tags"] = {**(existing_tags), **new_tags}
 
         return pulumi.ResourceTransformationResult(args.props, args.opts)
-
-
-# # auto_tag applies the given tags to the resource properties if applicable.
-# def auto_tag(args, auto_tags):
-#     if is_taggable(args.type_):
-#         # Ensure that the "tags" property is set in args.props
-#         existing_tags = args.props.get("tags", {}) or {}
-
-#         # Check if auto_tags is not None and is a dictionary before processing
-#         if auto_tags and isinstance(auto_tags, dict):
-#             print(f"Auto-tagging {args.type_}")
-#             # Check if any of the auto_tags already exist in the resource's tags
-#             new_tags = {k: v for k, v in auto_tags.items() if k not in existing_tags}
-#             args.props["tags"] = {**(existing_tags), **new_tags}
-
-#             return pulumi.ResourceTransformationResult(args.props, args.opts)
+    # If you want to test to see if resources are not taggable, you can run a pulumi preview, and uncomment this line.
```

Comment:
Rather than have these in a commented thing, I would prefer we do an `if some_debug_condition` and make it live reachable code. Maybe an ENV var or something like that. `os.getenv('DEBUG', False)` perhaps?

Reply:
i made the change, and tested it.